MyCryptoHQ/MyCrypto
# Pre-filled Transaction Implementation for Send-Assets Flow

### Description

Allows a user to create a pre-filled transaction by including data in the URL.

### Changes

* Added in transaction form data initialization handling for pre-filled transaction

### Steps to Test

1. Navigate to http://localhost/send?gasPrice=50&gasLimit=50000&sendMode=token&tokenSymbol=OMG&value=50&data=0x0asd121625&to=0x80200997f095da94E404F7E0d581AAb1fFba9f7d

2. This should auto-populate many fields.
